NETProvider/src/FirebirdSql.Data.UnitTests/Properties/AssemblyInfo.cs

```diff
@@ -22,7 +22,7 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
-[assembly: CLSCompliant(true)]
+[assembly: CLSCompliant(false)]
```

Comment:
And I don't think that setting CLSCompliant to false for the unit-test changes anything.